docs/index.md

```diff
@@ -248,6 +248,37 @@ To facilitate this workflow, the issuer can include a link to help the user down
   ]
 }
 ```
+#### on Android
+When issuing a SMART Health Card from one app to another on Android, use:
```

Comment:
I'd probably change "issuing" to "transferring" or "sending" to clarify that the sender is not an issuer in the SHC sense.

Reply:
```suggestion
When transferring a SMART Health Card from one app to another on Android, use:
```